docs/core/usage.qmd

```diff
@@ -466,6 +465,7 @@ column         | type     | unit     | restriction
 node_id        | Int      | -        | sorted
 active         | Bool     | -        | (optional, default true)
 listen_node_id | Int      | -        | -
+target_level   | Float64  | $m$      | -
```

Comment:
Is optional

Reply:
How can it be optional? The PID control cannot function without a setpoint. For the Basin it was optional because not all basins have a target level (so before it was required that the PID control node was connected to a basin with a target_level).